install.sh

```diff
@@ -306,7 +306,7 @@ nvm_do_install() {
     command printf "${SOURCE_STR}"
   else
     case "${NVM_PROFILE-}" in
-      ".bashrc" | ".bash_profile" | ".zshrc")
+      *"/.bashrc" | *"/.bash_profile" | *"/.zshrc")
```

Comment:
Right - so let's refactor out the detection into a separate function, and make tests just for it.